scaleway/resource_object_bucket.go

```diff
@@ -275,6 +281,13 @@ func resourceScalewayObjectBucketDelete(ctx context.Context, d *schema.ResourceD
 		return diag.FromErr(err)
```

Comment:
The delete is a new topic for this resource. Please check if the error is implemented as [here](https://github.com/hashicorp/terraform-provider-aws/blob/26f2311f79416f5d37c439288a7287a3b6a56c26/internal/service/s3/bucket.go#L1288).

Reply:
Please check if bucket exist like [this](https://github.com/hashicorp/terraform-provider-aws/blob/26f2311f79416f5d37c439288a7287a3b6a56c26/internal/service/s3/bucket.go#L1284)